packages/form-js-viewer/test/spec/render/components/form-fields/Dynamiclist.spec.js

```diff
@@ -75,6 +75,21 @@ describe('Dynamic List', () => {
     expect(collapseButton).to.exist;
   });
 
+  it('should render collapsed items under `fjs-repeat-row-collapsed`', function () {
+    // given
+    const field = {
+      ...defaultField,
+      nonCollapsedItems: 1,
+    };
+
+    // when
+    const { container } = createDynamicList({ field });
+
+    // then
+    const collapsedItems = container.querySelectorAll('.fjs-repeat-row-collapsed .fjs-form-field-textfield');
+    expect(collapsedItems).to.have.length(2);
+  });
```

Comment:
This test is not really guaranteeing that we're outputting the correct data, the ideal test would be to just check the output data. It it possible to write a test like that?

I would just delete this test if it's not possible since this test is just testing the implementation details of the dynamic list